lnrpc/walletrpc/walletkit.proto

```diff
@@ -234,6 +239,13 @@ message AddrResponse {
     string addr = 1;
 }
 
+message AddrsResponse {
+    /*
+    Array of addresses encoded using a bech32 format.
+     */
```

Comment:
nit: extra space here

Reply:
You can also run `make rpc-format` to automatically clean this up.

Reply:
Running `make rpc-format` doesn't seem to fix this -_- 

I'll just make the change manually